ui/appui-abstract/src/appui-abstract/UiItemsManager.ts

```diff
@@ -13,8 +13,14 @@ import { CommonStatusBarItem } from "./statusbar/StatusBarItem";
 import { CommonToolbarItem, ToolbarOrientation, ToolbarUsage } from "./toolbars/ToolbarItem";
 import { AbstractWidgetProps } from "./widget/AbstractWidgetProps";
 import { AbstractZoneLocation, StagePanelLocation, StagePanelSection } from "./widget/StagePanel";
-import { loggerCategory } from "./utils/misc";
 import { UiItemsProvider } from "./UiItemsProvider";
+import { getClassName } from "./utils/misc";
+
+const loggerCategory = (obj: any): string => {
+  const className = getClassName(obj);
+  const category = `appui-abstract${(className ? `.${className}` : "")}`;
+  return category;
+};
```

Comment:
This keeps using the newly deprecated `getClassName` function, and is used only to generate the string `appui-abstract.UiItemsManager`, might as well just replace the string :)

Reply:
Thanks - that makes sense to me.